CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+cmake_minimum_required(VERSION 3.10)
+
+project(ExaMiniMD LANGUAGES CXX)
+
+find_package(Kokkos 3.0 REQUIRED)
+option(USE_MPI "Build with MPI" ON)
+
+if (USE_MPI)
+  find_package(MPI REQUIRED CXX)
+else()
+  kokkos_check(DEVICES Cuda RETURN_VALUE USE_CUDA)
+  if (USING_CUDA)
+    if (CMAKE_CXX_COMPILER_ID MATCHES "nvcc")
```

Comment:
@dalg24 is there a better way to do this?

Reply:
not that I know of